models/Host.v1.yaml

```diff
@@ -1,5 +1,5 @@
 title: Host
-description: A client host assigned to a Signin
+description: "The root of the Host type's schema."
```

Comment:
Should revert this file's changes
```suggestion
description: A client host assigned to a Signin
```

Reply:
Is `A client host assigned to a Signin` an appropriate description for a Host?